src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/entities/Tasks.java

```diff
@@ -0,0 +1,177 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.cft.entities;
+
+import com.vladmihalcea.hibernate.type.basic.PostgreSQLEnumType;
+import com.vladmihalcea.hibernate.type.json.JsonType;
+import lombok.Getter;
+import lombok.ToString;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.BusinessContext;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.TaskState;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.TaskSystem;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.utils.Notes;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.entities.camunda.SecurityClassification;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+
+@ToString
+@Getter
+@Entity
+@TypeDef(
+    name = "pgsql_enum",
+    typeClass = PostgreSQLEnumType.class
+)
+@TypeDef(
+    name = "jsonb",
+    typeClass = JsonType.class
+)
+@SuppressWarnings({"PMD.ExcessiveParameterList", "PMD.TooManyFields"})
+public class Tasks implements Serializable {
```

Comment:
We probably need to review the naming.  The table is called `Tasks` because it holds lots of Tasks but this represents a single `Task` with a unique id.
However, we have other objects using the name `Task` within the domain so would be nice if they didn't clash.  So could possibly name this `CftTask` or `TaskResource` or `TaskRecord` ?